test/test_translator.py

```diff
@@ -228,31 +308,39 @@ def test_get_event_status():
 
-    # info={}
-    # info['feed_info_id'] = "feed_info_id"
-    #
-    # #### This information is required, might want to hardcode
-    # info['metadata'] = {}
-    # info['metadata']['wz_location_method'] = "channel-device-method"
-    # info['metadata']['lrs_type'] = "lrs_type"
-    # info['metadata']['location_verify_method'] = "location_verify_method"
-    # info['metadata']['datafeed_frequency_update'] = 86400
-    # info['metadata']['timestamp_metadata_update'] = "timestamp_metadata_update"
-    # info['metadata']['contact_name'] = "contact_name"
-    # info['metadata']['contact_email'] = "contact_email"
-    # info['metadata']['issuing_organization'] = "issuing_organization"
-    test_wzdx=icone_translator.wzdx_creator(icone_obj,icone_translator.initialize_info())
 
-    validate(instance=test_wzdx,schema=wzdx_schema)
+  test_wzdx = icone_translator.wzdx_creator(icone_obj, icone_translator.initialize_info())
+  print(json.dumps(test_wzdx))
 
+  assert re.match(wzdx_re,json.dumps(test_wzdx)) != None
+
+def test_wzdx_creator_empty_icone_object() :
+
+  icone_obj = None
+  test_wzdx = icone_translator.wzdx_creator(icone_obj)
+  assert test_wzdx == None
 
 def test_parse_arguments():
```

Comment:
Good success test case, but what about when we have no arguments, or invalid arguments, or the parsing of them fails?

Reply:
i added some edge cases according to your comment..and updated to dev

Reply:
This looks good, the only case I still see to test is passing `-h` to cause the `sys.exit()` to be called.

Reply:
i added the unit test for help